docs/75 LeetCode Questions/Arrays&Strings/5. Reverse Vowels of a String.md

```diff
@@ -0,0 +1,55 @@
+---
+id: reverse-vowels-of-a-string-345
+title: Reverse Vowels of a String
+sidebar_label: Leetcode 345
+tags: [Leetcode, String, DSA, Reverse Vowels of a String]
+description: Given a string s, reverse only all the vowels in the string and return it.The vowels are 'a', 'e', 'i', 'o', and 'u', and they can appear in both lower and upper cases, more than once.
```

Comment:
If you are using single quotes (`'`), then you need to write your string content in double quotes (`"`) with a close whole sentence. That way, the system can easily understand your code.